packages/dht/src/connection/WebSocket/WebSocketConnector.ts

```diff
@@ -31,32 +31,31 @@ export class WebSocketConnector implements IWebSocketConnectorService {
     private readonly rpcCommunicator: ListeningRpcCommunicator
     private readonly canConnectFunction: (peerDescriptor: PeerDescriptor, _ip: string, port: number) => boolean
     private readonly webSocketServer?: WebSocketServer
-    private readonly connectivityChecker: ConnectivityChecker
+    private connectivityChecker?: ConnectivityChecker
     private readonly ongoingConnectRequests: Map<PeerIDKey, ManagedConnection> = new Map()
     private incomingConnectionCallback: (connection: ManagedConnection) => boolean
-    private webSocketPort?: number
+    private webSocketPortRange?: PortRange
```

Comment:
```suggestion
    private portRange?: PortRange
```

(could drop the prefix from `webSocketServer` field)